### DISCUSSION TITLE
Frodo CLI 2.0.0 Ready?

### DISCUSSION THREAD

**Discussion by vscheuber** [rockcarver/frodo-cli#339] [Polls]
Frodo CLI 2.x has been in pre-release for months with awesome enhancements and plenty of bug fixes but one of the biggest changes is certainly the addition of e2e tests.

**Comment by vscheuber** [comment]
@atomicsamurai @hfranklin @phalestrivir ^^

**Reply by atomicsamurai** [reply]
I would like add the --no-deps flag to SAML import/export before we do it. Can we wait till the new year? :)

**Reply by phalestrivir** [reply]
I agree with @atomicsamurai, I think we should wait until the new year. I have a couple other features I would like to add in that I'm currently working on that should be finished this week.

**Comment by vscheuber** [comment]
I'm happy to delay into the new year. Let's just make sure we don't sacrifice better for best. Even if we release today, we can roll out updates like we've always done. The big thing that will change with the release is that everybody will start getting the notification where as currently 2.x is a stealth branch.

**Comment by vscheuber** [comment]
@phalestrivir @atomicsamurai @hfranklin Please cast your vote if you think 2.0.0 is a go from a feature/stability perspective. We still need to brush up the README.md and other docs and the CHANGELOG.md.

We were waiting on @atomicsamurai 's SAML dependency PR and it just came in and as now in 2.0.0-44.

Let's vote! :)

**Reply by hfranklin** [reply]
Generally, 2.0 seems like it is in a good place. I can't think of anything reasonable we can get it more ready to release. My vote is to release it after the holiday on Monday.
